### PR TITLE
add api and 'terms and conditions' redirects

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- add api and 'terms and conditions' redirects

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- [PLATOPS-44](https://airqoteam.atlassian.net/browse/PLATOPS-44?atlOrigin=eyJpIjoiZWNjYjYxNzliMjMyNDhhMDkzZThmMjE5YjQyNDE2NmQiLCJwIjoiaiJ9)

**_HOW DO I TEST OUT THIS PR?_**
- Go to https://staging-docs.airqo.net/airqo-rest-api-documentation
- You will automatically be redirected to the path https://staging-docs.airqo.net/#/api/
- Go to https://staging-docs.airqo.net/airqo-terms-and-conditions/HxYx3ysdA6k0ng6YJkU3
- You will automatically be redirected to the path https://staging-docs.airqo.net/#/platform/terms_and_conditions

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_**
- /airqo-rest-api-documentation
- /airqo-terms-and-conditions/HxYx3ysdA6k0ng6YJkU3

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- [This frontend PR](https://github.com/airqo-platform/AirQo-frontend/pull/956)

### CODE REVIEW

**Comment by 123MwanjeMike** [comment]
Note that this PR **depends** on [this frontend PR](https://github.com/airqo-platform/AirQo-frontend/pull/956)

File: k8s/nginx/production/global-config.yaml [hunk 1]
```diff
@@ -18,7 +18,9 @@ data:
     location @access_denied { return 403 '{\"success\":false, \"message\":\"Access denied :|\", \"error\":\"Insufficient Permissions\"}'; }
     location @unauthorized { return 401 '{\"success\":false, \"message\":\"Invalid authentication\", \"error\":\"Unauthorized.\"}'; }
 
+    location /airqo-rest-api-documentation { return 302 /#/api/; }
```
Comment (123MwanjeMike) [inline k8s/nginx/production/global-config.yaml:21]: API redirect

File: k8s/nginx/production/global-config.yaml [hunk 2]
```diff
@@ -18,7 +18,9 @@ data:
...
     location /airqo-application-documentations { return 302 /#/mobile_app/privacy_policy; }
+    location /airqo-terms-and-conditions/HxYx3ysdA6k0ng6YJkU3 { return 302 /#/platform/terms_and_conditions; }
```
Comment (123MwanjeMike) [inline k8s/nginx/production/global-config.yaml:23]: Terms and conditions redirect

**Review by Baalmart — APPROVED** [review]
Brilliant! Thanks @123MwanjeMike

**Pull request merged**